### ISSUE TITLE
Not Posting Encrypted Content

### ISSUE THREAD

**Issue by derekirving** [kennethmyhra/Http.EncryptedContentEncoding#1]
https://github.com/kennethmyhra/Http.EncryptedContentEncoding/blob/98b5f337a6c8935ce75fc1415367133fc3ff3dde/Client/Program.cs#L80

Thanks for building this example, it's been really helpful to me but your are not sending encrypted content in the `HttpPostEncryptedContentAndReceiveEncryptedResponseAsync` method as the name suggests. I think the method should be:

```
using (var client = new HttpClient())
{
    HttpContent content = new StringContent(data, Encoding.UTF8, contentType);
    var key = KeyLocator(keyId);
    HttpContent encryptedContent = new Aes128GcmEncodedContent(content, key, keyId, 4096);

    client.DefaultRequestHeaders.Add("X-aes128gcm-KeyId", keyId);
    client.DefaultRequestHeaders.AcceptEncoding.Add(new StringWithQualityHeaderValue("aes128gcm"));
    content.Headers.ContentType = new MediaTypeHeaderValue(contentType);

    return await client.PostAsync(url, encryptedContent);
}
```

**Comment by kennethmyhra** [comment]
Thanks, sure looks like there is a mistake there, will look into it and apply your fix

**Comment by kennethmyhra** [comment]
Thanks again, Derek! Issue was fixed in commit: e30f44d. Also committed a couple of other fixes.

Make sure that you check out the library [Lib.Net.Http.EncryptedContentEncoding](https://github.com/tpeczek/Lib.Net.Http.EncryptedContentEncoding) by Tomasz Pęczek which this example uses.
